spec/tasks/send_post_deadline_reminders_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
```

Comment:
delete?

Reply:
https://www.mintbit.com/blog/exploring-frozen-string-literal-a-ruby-magic-comment#:~:text=The%20immutability%20enforced%20by%20frozen_string_literal,behavior%20in%20your%20Ruby%20code.